disputer/index.js

```diff
@@ -24,73 +24,76 @@ const ExpiringMultiParty = artifacts.require("ExpiringMultiParty");
  * @return None or throws an Error.
  */
 async function run(address, shouldPoll, pollingDelay, priceFeedConfig) {
-  Logger.info({
-    at: "Disputer#index",
-    message: "Disputer started🔎",
-    empAddress: address,
-    pollingDelay: pollingDelay,
-    priceFeedConfig
-  });
-
-  // Setup web3 accounts an contract instance
-  const accounts = await web3.eth.getAccounts();
-  const emp = await ExpiringMultiParty.at(address);
-
-  // Setup price feed.
-  // TODO: consider making getTime async and using contract time.
-  const getTime = () => Math.round(new Date().getTime() / 1000);
-  const priceFeed = await createPriceFeed(web3, Logger, new Networker(Logger), getTime, priceFeedConfig);
-
-  if (!priceFeed) {
-    throw "Price feed config is invalid";
-  }
+  try {
```

Comment:
moved the try-catch to encompass more than just the `while(true)` loop. Rather, the try-catch covers all logic in the `run` function.

Reply:
That's a good catch because there are errors that can be thrown when constructing the `disputer` for example, if the `default config` parameter is invalid